knplabs/knp-snappy-bundle/1.4/etc/packages/knp_snappy.yaml

```diff
@@ -0,0 +1,10 @@
+# Read Documentation: https://github.com/KnpLabs/KnpSnappyBundle/blob/v1.4/README.markdown
+knp_snappy: ~
+#    temporary_folder: %kernel.cache_dir%/snappy
+#    pdf:
+#        enabled:    true
+#        options: []
+#    image:
+#        enabled:    true
+#        options: []
```

Comment:
putting all available options commented does not make any sense:

- recipes are not about providing documentation
- if the bundle adds new config settings, this will get unmaintainable:
    - the recipe will not automatically mention them
    - even if the recipe gets updated, your existing project will not get an updated version of the skeleton file
- everyone using your project will have to wonder all the time about why all the config settings are commented